examples/headers/bpf_helper_defs.h

```diff
@@ -113,17 +113,17 @@ static __u64 (*bpf_ktime_get_ns)(void) = (void *) 5;
  *
  * 	This helper is a "printk()-like" facility for debugging. It
  * 	prints a message defined by format *fmt* (of size *fmt_size*)
- * 	to file *\/sys/kernel/debug/tracing/trace* from DebugFS, if
```

Comment:
Please don't update the headers, they are "vendored" from libbpf.

Reply:
Change reverted on last commit